linux/README.md

```diff
@@ -83,6 +83,26 @@ Nightly builds upload the most recent new master build to https://downloads.keym
 Jenkins now continuously builds Debian packages on every commit to master
 Periodically test packages will be uploaded to https://launchpad.net/~keymanapp/+archive/ubuntu/keyman-daily
 
+#### Launchpad
+1. If you don't have one, create an account at https://launchpad.net
+2. Request to join the ["Keyman for Linux"](https://launchpad.net/~keymanapp) team.
+3. Create a [GPG](https://help.ubuntu.com/community/GnuPrivacyGuardHowto) key and associate it to your launchpad account
+4. Set the following environment variables  
```

Comment:
I'd also mention to put those in your ~/.profile or ~/.bashrc so that you don't have to set them every time